### PR TITLE
Added index.js so that require('jmap-client') works. Fixes #33

### LINKED ISSUE #33: node import/require error
after npm install jmap-client

import jmap from 'jmap-client';
or
var jmap = require('jmap-client');

--------------------------- error
module.js:341
    throw err;
    ^

Error: Cannot find module 'jmap-client'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (E:\Temp\jmap-client\samples\cli\jmap-client-pass-cli.js:11:12)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)

### PR DESCRIPTION
**Pull request by ddolcimascolo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = exports = require('./dist/jmap-client.min.js');
```
Comment (dbenchi) [inline index.js:3]: A personal question: What is the diff between exports and module.exports? and why we should do both of them?

**Comment by dbenchi** [comment]
:+1:

**Comment by GrahamLinagora** [comment]
:+1:

File: index.js (hunk 1 above)
Comment (ddolcimascolo) [inline index.js:3]: `exports` starts as a reference to `module.exports`, but if you remap `module.exports` then `exports` is outdated. Thus the convention is to remap both as I did...

**Pull request merged**